go/tools/builders/read.go

```diff
@@ -198,6 +198,28 @@ func (r *importReader) findEmbed(first bool) bool {
 		case ' ', '\t':
 			// leave startLine alone
 
+		case '\'':
```

Comment:
optional: since we use `\'` more than once, consider extracting it to a constant